src/Core/SecretsManager/Commands/EnableAccessSecretsManager/EnableAccessSecretsManagerCommand.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Bit.Core.Entities;
+using Bit.Core.Exceptions;
+using Bit.Core.Repositories;
+using Bit.Core.SecretsManager.Commands.EnableAccessSecretsManager.Interfaces;
+
+namespace Bit.Core.SecretsManager.Commands.EnableAccessSecretsManager;
+
+public class EnableAccessSecretsManagerCommand : IEnableAccessSecretsManagerCommand
+{
+    private readonly IOrganizationUserRepository _organizationUserRepository;
+
+    public EnableAccessSecretsManagerCommand(IOrganizationUserRepository organizationUserRepository)
+    {
+        _organizationUserRepository = organizationUserRepository;
+    }
+
+    public async Task<List<(OrganizationUser, string)>> EnableUsersAsync(Guid organizationId,
```

Comment:
Please use named tuples, especially because the `string` property is not very descriptive (I had to look it up):

```suggestion
    public async Task<List<(OrganizationUser orgUser, string error)>> EnableUsersAsync(Guid organizationId,
```

Reply:
Thanks for taking a look! 

Resolved in https://github.com/bitwarden/server/pull/3020/commits/015c68aa31ccdeb0b246f7d3a903eb1312174b3d